src/register/registrationFields/CountryField.jsx

```diff
@@ -15,6 +15,14 @@ const CountryField = (props) => {
     if (props.onBlurHandler) { props.onBlurHandler({ target: { name: 'country', value } }); }
   };
 
+  const onBlurHandler = (event) => {
+    // Do not run validations when drop-down arrow is clicked
+    if (event.relatedTarget && event.relatedTarget.className.includes('pgn__form-autosuggest__icon-button')) {
+      return;
+    }
+    if (props.onBlurHandler) { props.onBlurHandler(event); }
```

Comment:
Should we rename the `props.onBlurHandler` to something different as its being used in both on blur and on click events?

Reply:
@zainab-amir Yes, you are right, it's being used for both `onBlur` and `onSelected` and the reason is we want to validate the input on both `onBlur` and `onSelected` events. And in authn, we know that validations are run by the `onBlur` event. Therefore I think it makes sense. 

But if you have suggestions for a name. please let me know.